src/main/java/com/google/api/codegen/config/ProtoField.java

```diff
@@ -41,7 +42,11 @@
   public ProtoField(Field protoField) {
     Preconditions.checkNotNull(protoField);
     this.protoField = protoField;
-    this.protoTypeRef = ProtoTypeRef.create(protoField.getType());
+    TypeRef typeRef = protoField.getType();
+    if (typeRef == null) {
```

Comment:
What case is this new condition handling? Could (should) protoField.getType() ever by null?

Reply:
Hmm I thought that this was necessary some time ago, but I just ran tests without this change and it's fine. Reverted.